tests/FondOfSpryker/Zed/PersistentCart/Business/Model/QuoteDeleterTest.php

```diff
@@ -1,157 +0,0 @@
-<?php
```

Comment:
Why?

Reply:
Could not solve the error in the partner shop